lib/connector.js

```diff
@@ -218,6 +218,62 @@ Connector.prototype.id = function(model, prop) {
   return p && p.id;
 };
 
+/**
+ * Return the database mapping name of the property of the model if it exists.
+ * Otherwise return the property name.
+ * Some connectors allow the column/field name to be customized
+ * at the model property definition level as `column`,
+ * `columnName`, or `field`. For example,
+ *
+ * ```json
+ * "name": {
+ *   "type": "string",
+ *   "mysql": {
+ *     "column": "NAME"
+ *   }
+ * }
+ * ```
+ * @param {String} model The target model name
+ * @param {String} prop The property name
+ *
+ * @returns {String} The database mapping name of the property of the model if it exists
+ */
+Connector.prototype.propertyMapping = Connector.prototype.column =
```

Comment:
+1 to keep `column` for backwards compatibility.

I have mixed feelings about `propertyMapping` name, I would expect "mapping" to provide the mapping rules to apply, not the final value. Also it's a good practice to start method and function names with a verb, e.g. `getSomething`.

Few alternative names to consider:
- `getPropertyName`
- `getPropertyKey`
- `mapPropertyName`
- `mapPropertyToDbField`
- `mapPropertyToDbFieldName`
- `getPropertyDbName`